telegram-notifier/LambdaFunction.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import print_function
+
+import boto3
+import json
+import logging
+import os
+import urllib
+from urllib2 import Request, urlopen, URLError, HTTPError
+
+"""
+BOT_ID = Your Telegram BOT ID. Example: bot123456789:XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
+CHAT_ID = Is your channel ID or your BOT chat window id
+"""
+
+BOT_ID = "bot123456789:XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
+CHAT_ID = "01234567891234"
+API_ENDPOINT = "https://api.telegram.org/%s/sendMessage" % BOT_ID
```

Comment:
As a best practice configuration should be stored in environment variables instead of inline. This will also ease parameterizing the config in a cloudformation template